movement.cpp

```diff
@@ -0,0 +1,435 @@
+#include "movement.h"
+#include "pixy.h"
+
+front_left_motor_ = Motor(frontLeftMotorPins[0], frontLeftMotorPins[1], frontLeftPwm);
```

Comment:
where is the movement class constrcutor?
this should be inside the constructor.